packages/cardpay-sdk/sdk/token-bridge-home-side.ts

```diff
@@ -229,11 +232,16 @@ export default class TokenBridgeHomeSide implements ITokenBridgeHomeSide {
         `Timed out waiting for tokens to be bridged to layer 2 for safe owned by ${recipientAddress} after block ${fromBlock}`
       );
     }
+    // TODO: can this use waitForSubgraphIndex?
```

Comment:
Can we answer or remove this question?

Reply:
I'm understanding this transaction to be the "minting" transaction on the layer 2 side, in the bridging process. It isn't directly user-initiated, so we don't currently have an easy way to get the transaction hash and wait for the subgraph to index that transaction. 

I've removed the comment without changing the existing implementation, please do correct me if I've misunderstood!